gtfs_realtime_translators/factories/factories.py

```diff
@@ -12,23 +12,27 @@ def create(entity_id, **kwargs):
 class TripUpdate:
 
     @staticmethod
-    def __get_stop_time_events(arrival_time, departure_time=None):
-        arrival = gtfs_realtime.TripUpdate.StopTimeEvent(time=arrival_time)
-        if departure_time is None:
-            departure = arrival
-        else:
+    def __get_stop_time_events(arrival_time, departure_time, arrival_delay, departure_delay):
```

Comment:
- I would preserve the logic for assigning departure = arrival in absence of departure
- Keep in mind, there were both arrival times and delays, this method would overwrite times in favor of delays. I think that should be ok for now, since it's unlikely a feed would publish both